script/libtest.bash

```diff
@@ -53,7 +53,7 @@ k8s_wait_for_pod_ready() {
 k8s_wait_for_uniq_pod() {
     echo_info "Waiting for pod '${@}' to be the only one running ... "
     local -i cnt=${TEST_MAX_WAIT_SEC:?}
-    until [ `kubectl get pod "${@}" | grep Running | wc -l | awk '{print $1}'` -eq "1" ] ; do
+    until [ `kubectl get pod "${@}" | grep 1/1 | wc -l | awk '{print $1}'` -eq "1" ] ; do
```

Comment:
Would below work?:
```
until [[ $(kubectl get pod "${@}" -ogo-template='{{.items|len}}') == 1 ]]; do 
```
If not, suggest:
```
until [[ $(kubectl get pod "${@}" |grep -c 1/1) == 1 ]]; do 
```

Reply:
still not working: if this gets executed before the updated post is created it detects the outdated one as the unique

Reply:
Would then suggest, from the added `test_kubeless_function_update()` function,
implement something like `wait_for_replaced_pod()` or alike, like e.g.:

```
local current_pod=$(kubectl get pod "${@}" -o name)
k8s_wait_for_replaced_pod "${current_pod}"
k8s_wait_for_pod_ready "${@}"
```
with `k8s_wait_for_replaced_pod` implemented with:
```
local cur_pod="${1:?}"
until [[ $(kubectl get pod "${@}" -o name) != ${cur_pod} ]]
```